components/NavBar/index.tsx

```diff
@@ -2,25 +2,24 @@ import { useCallback, useState } from 'react'
 import Link from 'next/link'
 import dynamic from 'next/dynamic'
 import NavLink, { NavLinkProps } from './NavLink'
+import Image from 'next/image'
```

Comment:
Try to group package imports and relative imports together so it becomes easier to find declarations when in bigger files. Also it tends to be convention in many companies